### PR TITLE
Centos and Origin support

### PR DESCRIPTION
**Pull request by cooktheryan**

### CODE REVIEW

File: reference-architecture/aws-ansible/README.md [hunk 1]
```diff
@@ -56,24 +69,45 @@ export AWS_SECRET_ACCESS_KEY=bar
...
 ```
+
+**OpenShift Origin**
+```
+./ose-on-aws.py --keypair=OSE-key --public-hosted-zone=sysdeseng.com --deployment-type=origin --ami=ami-6d1c2007
+```
+
 ### Existing AWS Environment (Brownfield)
-If the installing OpenShift into an existing AWS VPC perform the following. The script will prompt for vpc and subnet IDs.  The Brownfield deployment can also skip the creation of a Bastion server if one already exists. For mappings of security groups make sure the bastion security group is named bastion-sg.
+If the installing OpenShift Container Platform or OpenShift Origin into an existing AWS VPC perform the following. The script will prompt for vpc and subnet IDs.  The Brownfield deployment can also skip the creation of a Bastion server if one already exists. For mappings of security groups make sure the bastion security group is named bastion-sg.
```
Comment (themurph) [inline reference-architecture/aws-ansible/README.md]: Typo:
If **the** installing..

Missing Comma:
an existing AWS VPC, perform 

**A** Brownfield deployment can also

For mappings of security groups, make

File: reference-architecture/aws-ansible/README.md [hunk 2]
```diff
@@ -9,20 +9,33 @@ The repository contains Ansible playbooks which deploy 3 Masters in different av
 ## Prerequisites
 A registered domain must be added to Route53 as a Hosted Zone before installation.  This registered domain can be purchased through AWS.
 
-### OpenShift Playbooks
+### Deploying OpenShift Container Platform
```
Comment (themurph) [inline reference-architecture/aws-ansible/README.md]: Deploying **the** OpenShift Container Platform

Comment (detiber) [inline reference-architecture/aws-ansible/README.md]: Is the use of 'the' right here? Pulling in @adellape for his expert opinion :)

File: reference-architecture/aws-ansible/README.md [hunk 3]
```diff
@@ -56,24 +69,45 @@ export AWS_SECRET_ACCESS_KEY=bar
 The default region is us-east-1 but can be changed when running the ose-on-aws script by specifying --region=us-west-2 for example. The region must contain at least 3 Availability Zones. 
 
 ### AMI ID
-The AMI ID may need to change if the AWS IAM account does not have access to the Red Hat Cloud Access gold image or if deploying outside of the us-east-1 region.
+The AMI ID may need to change if the AWS IAM account does not have access to the Red Hat Cloud Access gold image, deploying CentOS, or if deploying outside of the us-east-1 region.
```
Comment (themurph) [inline reference-architecture/aws-ansible/README.md]: ~~deploying CentOS,~~
, another OS such as CentOs is deployed,

File: reference-architecture/aws-ansible/README.md (hunk 2 above)
Comment (cooktheryan) [inline reference-architecture/aws-ansible/README.md]: I"m from WV. I don't do English well so I will take either way with or without the "the"

**Comment by cooktheryan** [comment]
Fixes issue #56

**Pull request merged**